2200053/10505.py

```diff
@@ -0,0 +1,11 @@
+# 소득 불균형
+T = int(input())
+
+for t in range(1, T+1):
+    N = int(input())
+    num = list(map(int, input().split()))
+    cnt = 0
+    for n in num:
+        if n <= sum(num) // N:
```

Comment:
```suggestion
        if n <= sum(num) / N:
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄